applications/nrf_desktop/README.rst

```diff
@@ -955,6 +955,42 @@ To use the nRF Desktop application with your custom board:
    Some options have dependencies that might not be needed when these options are disabled.
    For example, when the LEDs module is disabled, the PWM driver is not needed.
 
+Adding a new motion sensor
+==========================
+
+The nRF52 Desktop application comes with a motion module able to read data from a motion sensor.
+A communication with the sensor is done through a sensor driver API.
+In the scope of NCS two motion sensor drivers are available, for PMW3360 and PAW3212.
+It is likely that another sensor will be more suitable for a new board.
+In this section it is described how to add a new motion sensor into the project.
+
+The first step is to create a new motion sensor driver, that will provide code communicating with the sensor.
+Driver shall provide the sensor API (see :ref:`sensor_api`), that will be used by the application.
+The two existing drivers can be used as an example.
+For motion module to correctly work with a driver, it must support a trigger on a new data.
+When data is ready the motion module will call data fetch and data get on two sensor channels ``SENSOR_CHAN_POS_DX`` and ``SENSOR_CHAN_POS_DY``.
```

Comment:
`When data is ready` - can we be more specific here?

Reply:
I have changed it to "When motion data is ready".